test/gtest/uct/ib/test_ib_md.cc

```diff
@@ -310,4 +310,22 @@ UCS_TEST_P(test_ib_md, smkey_reg_atomic_mt, "IB_REG_MT_THRESH=1k",
     test_smkey_reg_atomic();
 }
 
+UCS_TEST_P(test_ib_md, reg_advise_odp_no_devx, "IB_MLX5_DEVX=no")
+{
+    ASSERT_EQ(md_attr().reg_nonblock_mem_types, UCS_BIT(UCS_MEMORY_TYPE_HOST));
+    for (auto size: {UCS_KBYTE, UCS_MBYTE}) {
+        test_reg_advise(size, size, 0, true);
+        test_reg_advise(size, size / 2, 0, true);
+        test_reg_advise(size, size / 2, size / 4, true);
+    }
```

Comment:
maybe move this common part to a helper function in test_md.h/.cc?

Reply:
Done